### ISSUE TITLE
Don't show the ontology pop up when setting a decimal unit

### ISSUE THREAD

**Issue by homonecloco** [collaborative-open-plant-omics/COPO#8]
If the unit is numeric, there is no need to show the ontologies. Furthermore, it may lead to mistakenly clicking the name of an ontology instead of a number.

**Comment by homonecloco** [comment]
In the tooltips, it may be useful to have the values that had been entered before in the experiment. For example, I have replicates at Zadoks (the growth stage unit) 11, 35, 60, 64, etc. As this are consistent time points, it showing which ones had been captured may reduce the chances of typos (34 is valid, but if it hasn't been entered don't suggest it, for example).

**Comment by shaw2thefloor** [comment]
Since there are many ontology terms which start with numbers, it is not reasonable to assume that numeric input is not a term specifier.

**Comment by homonecloco** [comment]
I mean for the ontologies that have values as numbers. The example of Zadoks, is one that is based totally numeric. But a 'clear' example would be centimetres. If I select centimetres as unit, the value will decently not be an ontology.

**Comment by shaw2thefloor** [comment]
Sure. But if you type '12' as 12 cm, how does the system know that you didn't mean '12-ethyl-8-propyl-3-vinyl-bacteriochlorophyllide d(1-)'.

**Comment by homonecloco** [comment]
You have your field UNIT. If you already set that, and you put '12-ethyl-8-propyl-3-vinyl-bacteriochlorophyllide d(1-)', it is not centimetres and it should validate that it is not a valid value for centimetres (I assume the ontology of centimetres is setup as a numeric value).

**Comment by shaw2thefloor** [comment]
So you would want to have unit before value? i.e. Category - Unit - Value. I'm not sure thats very intuitive.

**Comment by homonecloco** [comment]
That is a good point. Usually the Unit would go after the value. So I don't think the order in the interface would change. However, if you are cloning samples, the units would be already filled. So, if the unit is already set, you could go trough the extra validation of numeric/non-numeric values. Does that make sense?

**Comment by shaw2thefloor** [comment]
Sure it makes sense, but I don’t know if we want to change the ordering of the fields. Its very important to make things as intuitive as possible.


On 5 Feb 2018, at 12:08, Ricardo H. Ramírez-Gonzalez <notifications@github.com<mailto:notifications@github.com>> wrote:


That is a good point. Usually the Unit would go after the value. So I don't think the order in the interface would change. However, if you are cloning samples, the units would be already filled. So, if the unit is already set, you could go trough the extra validation of numeric/non-numeric values. Does that make sense?

**Comment by homonecloco** [comment]
I agree on not changing the order of the fields. That is fine as it is.

**Comment by shaw2thefloor** [comment]
So you would just manually set the unit first and then the value? Obviously the tabbing won’t work for this unless we were to set an option for that somewhere.

On 5 Feb 2018, at 12:11, Ricardo H. Ramírez-Gonzalez <notifications@github.com<mailto:notifications@github.com>> wrote:


I agree on not changing the order of the fields. That is fine as it is.

**Comment by homonecloco** [comment]
I think if you are doing a new sample, the current behaviour is fine. It is when you are editing a sample where you can add the validation. As I said in a previous comment, it would be when preparing a large batch of samples with the same fields.

**Comment by shaw2thefloor** [comment]
Ok thanks for the input Ricardo. Do you by any chance have a spreadsheet with your sample information in it? We are going to allow to inputting of samples by uploading a spreadsheet quite soon, so this may get round your problems.
